funlet-forward/tests/funlet-forward.test.js

```diff
@@ -73,73 +71,40 @@ const IS_RESTRICTED_NUMBER = false;
 const XML_DECLARATION = '<?xml version="1.0" encoding="UTF-8"?>';
 
 const FULL_RESPONSE_FORWARD_1_1 =
-  XML_DECLARATION +
-  '<Response>' +
-  '<Dial ' +
-  'action="' +
-  BASE_ACTION_URL +
-  '" ' +
-  'timeout="' +
-  DEFAULT_TIMEOUT +
-  '"' +
-  '>' +
-  PHONE_NUMBER +
-  '</Dial>' +
-  '</Response>';
+  `${XML_DECLARATION}<Response>` +
```

Comment:
all these multiline strings can be written to use a single starting \` and an ending `